.mergify.yml

```diff
@@ -10,6 +10,12 @@ rules:
         strict: true
         contexts:
           - ci/jenkins/pr_tests
+          - crate.crate
```

Comment:
what's this?

Reply:
Azure pipelines, running `:sql:test` on both windows and linux hosts. We set it up yesterday